src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate(operation, number) {
+      this.result = operation(this.result, number);
+
+      return this;
+    },
+    add(total, number) {
+      return number + total;
+    },
```

Comment:
for every operation you can interact directly with `this.result` and a `number` - so you'll need only one parameter
```suggestion
    add(number) {
      this.result += number;
    },
```

Reply:
Firstly I tried with one parametr but got `this` as undefined, 
forgot `this` depends on context runtime 
so each operation method don't have access to `this`